### PR TITLE
Fix Release 10.5.z

### PR DESCRIPTION
**Pull request by Artiume**
**Changes**

**Issues**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,8 +4,8 @@ ARG FFMPEG_VERSION=latest
 FROM node:alpine as web-builder
 ARG JELLYFIN_WEB_VERSION=10.5.1
 RUN apk add curl git \
- && curl -L https://github.com/jellyfin/jellyfin-web/archive/${JELLYFIN_WEB_VERSION}.tar.gz | tar zxf - \
- && cd jellyfin-web-* \
+ && git clone --branch release-10.5.z --single-branch https://github.com/jellyfin/jellyfin-web.git \
```
Comment (joshuaboniface) [inline Dockerfile]: We can't hardcode this stuff in. It has to be fully dynamic, or it will break boht nightlies and future releases.

**Comment by Artiume** [comment]
This is only on the 10.5.z branch, not master

**Comment by joshuaboniface** [comment]
Somehow I missed that. OK, it's a worthwhile compromise for 10.5.0 and with 10.6.0 we can finally fix out packaging to not be so obtuse.

**Comment by joshuaboniface** [comment]
If it works (i.e. builds), might as well just add all the other fixes to the same PR. Then I can test them *en masse* later this week.

**Comment by Artiume** [comment]
No idea

File: Dockerfile (hunk 1 above)
Comment (mark-monteiro) [inline Dockerfile]: Might be worthwhile to add the `--depth 1` option here to avoid pulling more git history than necessary

Comment (Artiume) [reply Dockerfile]: Could that cause issues? And not sure if it matters much, this is only going to be for point releases

**Pull request merged**